src/OpenColorIO/GpuShaderUtils.cpp

```diff
@@ -2,14 +2,16 @@
 // Copyright Contributors to the OpenColorIO Project.
 
 #include <math.h>
+#include <vector>
+#include <iomanip>
+#include <string>
 
```

Comment:
Are these includes still needed?

Reply:
Last two can be removed. But vector is still needed.